Artifactory Plugin publish

eaiesb opened the pull request:
Artifactory Plugin publish

jyotisingh commented on data/task_plugins.yml:
```diff
@@ -204,3 +204,12 @@
   github_stars: http://ghbtns.com/github-btn.html?user=insano10&repo=gocd-pr-comment-scalastyle&type=watch&count=true
   first_available_date: 05/2016
+  
+- name:: Artifactory Plugin
```
There are two colons here`::`, should be just one

eaiesb commented:
Hi

Extra : removed and updated. Please let me know if you need any

 

Thank You

Much Respect

Vijaya Kumar Reddy.Maddela

 <mailto:Vijay.Reddy@eaiesb.com> Vijay.Reddy@eaiesb.com

91-40-40119993|9704179993

USA:978-306-3751



 

From: Jyoti Singh [mailto:notifications@github.com] 
Sent: 31 May 2017 15:30
To: gocd/www.go.cd <www.go.cd@noreply.github.com>
Cc: eaiesb <vijay.reddy@eaiesb.com>; Author <author@noreply.github.com>
Subject: Re: [gocd/www.go.cd] Artifactory Plugin publish (#534)

 

@jyotisingh commented on this pull request.

  _____  

In data/task_plugins.yml <https://github.com/gocd/www.go.cd/pull/534#discussion_r119318571> :

> @@ -204,3 +204,12 @@
   releases_url: https://github.com/insano10/gocd-pr-comment-scalastyle/releases
   github_stars: http://ghbtns.com/github-btn.html?user=insano10 <http://ghbtns.com/github-btn.html?user=insano10&repo=gocd-pr-comment-scalastyle&type=watch&count=true> &repo=gocd-pr-comment-scalastyle&type=watch&count=true
   first_available_date: 05/2016
+  
+- name:: Artifactory Plugin

There are two colons here::, should be just one

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/gocd/www.go.cd/pull/534#pullrequestreview-41184888> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AIqNGsgGJk8Ghw4-drWMhX12BRtawbocks5r_To4gaJpZM4NUCPM> .  <https://github.com/notifications/beacon/AIqNGrmqCsFxSJ165yJPW93-dNoaihSjks5r_To4gaJpZM4NUCPM.gif>
